data/openstreetmaps/src/main/java/de/mm20/launcher2/openstreetmaps/OsmSerialization.kt

```diff
@@ -0,0 +1,21 @@
+package de.mm20.launcher2.openstreetmaps
+
+import de.mm20.launcher2.search.SavableSearchable
+import de.mm20.launcher2.search.SearchableDeserializer
+import de.mm20.launcher2.search.SearchableSerializer
+
+class OsmLocationSerializer : SearchableSerializer {
+    override fun serialize(searchable: SavableSearchable): String {
+        searchable as OsmLocation
+        TODO()
```

Comment:
TODO

Reply:
relatively simple, just save the unique ID of the location